core/sql/sqlcomp/CmpSeabaseDDLcommentOn.cpp

```diff
@@ -289,6 +289,31 @@ void  CmpSeabaseDDL::doSeabaseCommentOn(StmtDDLCommentOn   *commentOnNode,
 
   ExeCliInterface cliInterface(STMTHEAP, NULL, NULL,
                                        CmpCommon::context()->sqlSession()->getParentQid());
+  //if comment on native HBASE first shoud register first
+  if (CmpSeabaseDDL::isHbase(catalogNamePart))
+  {
+      // HBase."MAP" fromater should covert
+      if (ComIsHBaseMappedExtFormat(catalogNamePart, schemaNamePart))
+      {
+           ComConvertHBaseMappedExtToInt(catalogNamePart, schemaNamePart, catalogNamePart, schemaNamePart);
+      }
+      else
+      {
+          //HBASE erternal table already register in create time no need to register again
+          NAString ddl = "REGISTER INTERNAL HBASE TABLE IF NOT REGISTERED ";
+          ddl.append(extObjName.data());
+
+          Lng32 cliRC = cliInterface.executeImmediate(ddl.data());
+          // error code -8102 means the object already register
+          if (cliRC < 0 && cliRC != -8102)
```

Comment:
Since the ddl issued is 'register ...if not registered', an 8102 error will
not be returned if object already exists.
This check should not be done and all errors should be returned.